app/views/content/train-to-be-a-teacher.md

```diff
@@ -6,7 +6,7 @@ description: |-
 title_paragraph: |-
     If you’ve just graduated, you're studying for a degree or you want to change
     your career, we can help you understand what training you need to do to get
-    qualified teacher status (QTS). With QTS you get a minimum starting salary of £30,000.
+    qualified teacher status (QTS). With QTS you get a minimum starting salary of $salaries_qualifiedteacher_englandmin$.
```

Comment:
Needs a fix

Reply:
@martyn-w is this something you could look at pls? Not sure what's wrong with it!

Reply:
Hi @Sarah-DfE unfortunately this is going to require a codefix to resolve; the variables system isn't integrated inside of hero components (yet). Leave this with me.

Reply:
remove this change from this PR and do it separately (ask Martyn)